src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+import java.time.DateTimeException;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.util.*;
+
 public class Duke {
+    // Constants
+    final static String PARTITION = "*******************************************";
+    final static String EXIT_COMMAND = "bye";
+    final static String LIST_COMMAND = "list";
+    final static String DELETE_COMMAND = "delete";
+    final static String MARK_COMMAND = "mark";
+    final static String UNMARK_COMMAND = "unmark";
+    final static String TODO_COMMAND = "todo";
+    final static String EVENT_COMMAND = "event";
+    final static String DEADLINE_COMMAND = "deadline";
+
+    // Fields
+    private static List<Task> current_list = new ArrayList<>();
+    private static Path path;
+    private static File fileToRead;
+
+    // Methods
+    private static void checkInput(String[] current_input_array) throws EmptyDescriptionException {
+        if (current_input_array.length < 2) {
```

Comment:
The camelcase naming convention can be applied to the arguments of these functions as well